Gemfile.lock

```diff
@@ -348,4 +348,4 @@ DEPENDENCIES
   webmock
 
 BUNDLED WITH
-   1.13.6
+   1.14.5
```

Comment:
Sounds good. I wasn't sure if we had to have specific versions of bundler in CI etc, but it looks like it only gives a warning.

LGTM